### ISSUE TITLE
[18.0.3] Video player does not resize gifs to fit screen

### ISSUE THREAD

**Issue by Dragon988** [nextcloud/server#20116]
Using firefox.

Here is an example:

https://drive.irex.me/s/YSJt6SLcEXnMmXb

On a 1080p screen it cuts off the bottom portion of the GIF and does not resize the GIF to fit the screen.

**Comment by kesselb** [comment]
That looks like the wrong GIF :confused: Also please upload the GIF on GitHub ;)

**Comment by Dragon988** [comment]
I mean it's the correct gif. The video player isn't resizing it for me - see attached screenshot.

![firefox_eZdlyh4H3v](https://user-images.githubusercontent.com/7935624/77339189-a7b39c80-6ce8-11ea-8ab0-1338821ff690.png)

**Comment by kesselb** [comment]
All right. I was looking for the video player ;) But yeah the image preview. I would say that works as expected.

cc @nextcloud/designers @jancborchardt @skjnldsv

**Comment by Dragon988** [comment]
> 
> 
> All right. I was looking for the video player ;) But yeah the image preview. I would say that works as expected.
> 
> cc @nextcloud/designers @jancborchardt @skjnldsv

It's not supposed to resize the GIF to fit the screen resolution? If so maybe this should be a feature request.

**Comment by skjnldsv** [comment]
It's actually not a gif, previews are not animated in the shares if I recall correctly?

**Comment by Dragon988** [comment]
> 
> 
> It's actually not a gif, previews are not animated in the shares if I recall correctly?

It's a animated GIF. Here is a link:

https://drive.irex.me/s/YSJt6SLcEXnMmXb

The problem is if the GIF is larger then the screen resolution it does not resize to fit on the screen.

**Comment by skjnldsv** [comment]
Yeah, no, it is like other shares, it display the full image I think.
The gif is indeed displayed properly animated, I was mistaken :)

**Comment by Dragon988** [comment]
> 
> 
> Yeah, no, it is like other shares, it display the full image I think.
> The gif is indeed displayed properly animated, I was mistaken :)

Yeah. I'm not sure if this is an actual bug but I'd assume it should resize to fit the monitor resolution in the viewer. Seems like that would make the most sense.

**Comment by skjnldsv** [comment]
It is resized to fit the screen size, but is not dynamically resizing

**Comment by skjnldsv** [comment]
Confirmed then, images are resized if dev tools opened and narrowing the window.
Gifs are not

**Comment by szaimen** [comment]
Hi, please update to 24.0.8 or better 25.0.2 and report back if it fixes the issue. Thank you!
